package.json

```diff
@@ -43,5 +43,8 @@
     "eslint": "^5.0.0",
     "rimraf": "^2.6.1",
     "shelljs": "^0.7.7"
+  },
+  "eslintConfig": {
+    "extends": "./eslint.config.js"
```

Comment:
If we add the config like this, we should update the lint script:
https://github.com/cerner/eslint-config-terra/blob/master/package.json#L25

Reply:
Missed seeing this before the merged. Added in this commit: https://github.com/cerner/eslint-config-terra/commit/6265a4b245e07c3e8c7353d41d29d9c5e7f4ee34